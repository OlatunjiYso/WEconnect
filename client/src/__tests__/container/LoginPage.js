import React from "react";
import { shallow, mount, render } from 'enzyme';

import LoginContainer from '../../containers/login';

describe('All tests for login page', () => {
  
  const initialState = {};

  const store = mockStore(initialState);
  const wrapper = shallow(<Provider store={store}><LoginContainer/></Provider>);
  describe('it should render self', ()=>{
    it('it should render login page', () => {
      expect(wrapper.length).toEqual(1);
    });
    it('should match snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });
  }); 
});
