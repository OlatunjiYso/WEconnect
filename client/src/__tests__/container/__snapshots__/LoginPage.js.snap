// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All tests for login page it should render self should match snapshot 1`] = `<Connect(Login) />`;
